dev-proxy/OutdatedCommandHandler.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Microsoft.DevProxy;
+
+public static class OutdatedCommandHandler
+{
+    public static async Task CheckVersion()
+    {
+        var releaseInfo = await UpdateNotification.CheckForNewVersion(ProxyCommandHandler.Configuration.NewVersionNotification);
+        Console.WriteLine(releaseInfo?.Version);
```

Comment:
Let's use logger instead of writing directly to Console in case we introduce more/other loggers in the future